python/chronos/src/bigdl/chronos/forecaster/base_forecaster.py

```diff
@@ -66,7 +66,8 @@ def fit(self, data, epochs=1, batch_size=32):
                | should follow the shape stated before.
 
         :param epochs: Number of epochs you want to train. The value defaults to 1.
-        :param batch_size: Number of batch size you want to train. The value defaults to 32.
+        :param batch_size: Number of batch size you want to train. The value defaults to 32,
+               if 'distributed', the data divided by batch_size is evenly distributed to all node.
```

Comment:
if the forecaster is distributed, the batch_size will be evenly distributed to all workers.